packages/apps/human-app/frontend/src/modules/worker/jobs/my-jobs/components/desktop/my-jobs-expires-at-sort.tsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable camelcase --- ... */
 import { t } from 'i18next';
-import { useMyJobsFilterStore } from '@/modules/worker/hooks/use-my-jobs-filter-store';
 import { Sorting } from '@/shared/components/ui/table/table-header-menu.tsx/sorting';
+import { useMyJobsFilterStore } from '../../../hooks';
 
 export function MyJobsExpiresAtSort() {
```

Comment:
When calling `setFilterParams` in this and other places, do we really need to pass `...filterParams`? From what I see in `useMyJobsFilterStore` and `useJobsFilterStore` implementation - they already handle this